string first_string
string second_string
